autoload/ddc/ale.vim

```diff
@@ -1,17 +1,38 @@
-function! s:callback(plugin_name, method_name, results) abort
-  if type(a:results) != type([])
+let s:plugin_name = v:null
+let s:method_name = v:null
+
+function! s:callback(results) abort
+  if type(s:method_name) == v:t_none
```

Comment:
`v:t_none` doesn't seem to be supported in Neovim and causes an error. Could you get rid of all `v:t_none`s?
(I think we can use `type(v:null)` instead, but I haven't checked it.)

Reply:
Oops, didn't realize that, should be fixed now